src/app/client/src/app/modules/player-helper/components/content-player-metadata/content-player-metadata.component.html

```diff
@@ -36,7 +36,13 @@ <h4 class="sb-section-title text-capitalize mb-16 d-block">
     </dl>
     <dl class="sb-meta-data">
         <dt>{{resourceService.frmelmnts.lbl?.licenseTerms}}</dt>
-        <dd *ngIf="metadata.license">{{metadata.license}}&#x200E;</dd>
+        <dd *ngIf="metadata.license && typeof(metadata.license)">{{metadata.license}}&#x200E;</dd>
```

Comment:
can we use isString() or typeOf function should return any type like string object boolean etc.